### ISSUE TITLE
[Blog] Article entries allows editing

### ISSUE THREAD

**Issue by nvazquez** [apache/cloudstack-www#112]
When clicking on a blog article, the Edit page button is displayed (redirects to the repo and looks for the file on the main branch). Should this be allowed?

<img width="1155" alt="Screenshot 2023-05-09 at 22 26 46" src="https://github.com/apache/cloudstack-www/assets/5295080/67ce579b-d64b-4281-913f-16f9e49d5c47">

**Comment by rohityadavcloud** [comment]
Yup, this is part of the post - it redirect to the source blog post in the github repo. It could make it easier to edit pages (but we can explore if this needs to be disabled if Ivet advises).
